doc/extensions/acrbuildext/azext_acrbuildext/_format.py

```diff
@@ -0,0 +1,144 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
I am assuming this will be a copy of `_format.py` from `acr` folder.

Reply:
Right. It actually is a copy from src\command_modules\azure-cli-acr\azure\cli\command_modules\acr\_format.py
By the way, I will do another commit to reflect the updates I had just now. To let you know.

Reply:
Right. It actually is a copy from src\command_modules\azure-cli-acr\azure\cli\command_modules\acr\_format.py
By the way, I will do another commit to reflect the updates I had just now. To let you know.

Reply:
C:\git\src\github.com\AzureCR\azure-cli\src\command_modules\azure-cli-acr\azure\cli\command_modules\acr\_format.py

Reply:
C:\git\src\github.com\AzureCR\azure-cli\src\command_modules\azure-cli-acr\azure\cli\command_modules\acr\_format.py